orca-core/src/main/java/com/netflix/spinnaker/orca/notifications/scheduling/OldPipelineCleanupPollingNotificationAgent.java

```diff
@@ -169,7 +170,7 @@ private void cleanup(List<PipelineExecutionDetails> executions) {
     executions.subList(0, (executions.size() - minimumPipelineExecutions)).forEach(p -> {
       long startTime = p.startTime == null ? p.buildTime : p.startTime;
       long days = ChronoUnit.DAYS.between(Instant.ofEpochMilli(startTime), Instant.ofEpochMilli(clock.millis()));
-      if (days > thresholdDays && !executionRepository.hasEntityTags(PIPELINE, p.id)) {
```

Comment:
The `hasEntityTags` interface method no longer does anything. It was an interface added to support some Netflix functionality that never made it to OSS and has subsequently been removed from our extension. I removed it from OSS entirely as a result.